src/commands/__tests__/publish.test.js

```diff
@@ -94,7 +92,7 @@ describe('bolt publish', () => {
       }
     ]);
 
-    expect(publish({ cwd })).rejects.toBeInstanceOf(BoltError);
+    await expect(publish({ cwd })).rejects.toBeInstanceOf(Error);
```

Comment:
@lachlanhunt Just FYI, found why the test below was failing and also found that this test shouldn't have been passing. :S

We weren't awaiting on the expect.

There's another interesting thing here, we can't check for instanceOf(BoltError) because on lower versions of node it just comes through as "Error". Weird.